internal/opa/inspect_test.go

```diff
@@ -148,3 +157,75 @@ func TestInspectDir(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckRules(t *testing.T) {
+	cases := []struct {
+		name string
+		rego string
+		err  string
+	}{
+		{
```

Comment:
Let's add something like this for the missing test coverage:

```rego
deny := "value" {
	true
}
```

Reply:
Good catch!